python/lsst/daf/persistence/policy.py

```diff
@@ -250,6 +210,24 @@ def __contains__(self, key):
                 return False
         return keys[-1] in d
 
+    @staticmethod
+    def defaultPolicyFile(productName, filePath, repos=None):
+        """Get the path to a default policy file
+
+        @param productName (string) The name of the product that the default policy is installed as part of
+        @param filepath (string) The relative pathname to the policy file.  
+        @param repos (string) the subdirectory with the product's install directory where policy files are 
+                              stored. If None (default), the filepath argument is relative to the installation
+                              directory.
+        """
+        basePath = os.getenv(productName.upper() + "_DIR")
```

Comment:
this got cut & pasted from a code I wrote before I knew about getPackageDir. will fix.